docs/USERAPI.md

```diff
@@ -0,0 +1,206 @@
+# User API documentation
+
+This is the documentation for the new user API (and only the new one) that was created in 2020.
```

Comment:
Not sure what you mean by that.
Of this API is for new users only - please clarify
If this is User API v2, please use the term version (Today's "New" will get old someday :) )

Reply:
Yes, we can call it User API v2,
Somewhere between 2012 to 2014 someone wrote a User API system that is still present in the BE code and is also present in production, this code is currently broken.
To remove it we to do (a big?) refactor as this system is integrated into other parts of the server, and as there are future plans to rewrite all of BE server code - this task has currently been postponed. 

I will update the docs accordingly.